travis/clone_oca_dependencies

```diff
@@ -60,6 +60,11 @@ def git_checkout(deps_checkout_dir, reponame, url, branch):
         command = ['git', 'clone', '-q', url, '-b', branch, checkout_dir]
         _logger.info('Calling %s', ' '.join(command))
         subprocess.check_call(command)
+    else:
+        command = ['git', '--git-dir=' + os.path.join(checkout_dir, '.git'),
```

Comment:
For git commands used in runbot, like `gc`, `log` or `fetch`, which don't use a working tree, this works fine.

But `pull` uses a working tree, which is the current working directory by default.